src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,35 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
```

Comment:
```suggestion
    <li
```
use semantic tags, i.e `li` for list item (you're mapping `TodoInfo` components)